src/keywords.ts

```diff
@@ -5,4 +5,5 @@ export const sqlKeywords = [
 
     // added manually
     , "PRECISION"
+    , "SHARE"
```

Comment:
`SHARE` is not a keyword it is a regular name.

meaning that something like this is a valid statement:

```sql
create table share(key int)
```

If it were a keyword, this statement would throw a parse error.
For instance, `asymmetric` is a keyword... try executing:

```sql
create table asymmetric(key int);
```

... which makes me wonder why I added "precision" to the list of keywords (it is not). Dont remember, but it should not be here.

To use a name as a keyword, see [those](https://github.com/oguimbal/pgsql-ast-parser/blob/475071020d7a448a7200defe9ba4ce37fec24e42/src/syntax/base.ne#L128-L227)